CHANGELOG.md

```diff
@@ -1,3 +1,46 @@
+# 0.29 - 2022-07-20 "Edition 2018 Release"
+
+As promised, this is our quick release to bring on edition 2018 by increasing our MSRV to Rust
+1.41.1 [#983](https://github.com/rust-bitcoin/rust-bitcoin/pull/983)
+
+This work saw a bunch of new language features become available to us. At the
+same time we were able to start using `rustfmt`. We also started linting as part
+of CI.
+
+Some highlights:
```

Comment:
```suggestion
## Breaking changes

There are numerous breaking changes in this release related to the new language features but also other improvements such as more newtypes added. Note that not all changes cause compilation failure! [`Witness` serialization was changed](https://github.com/rust-bitcoin/rust-bitcoin/pull/1068) to support human-readable formats.

[Detailed list of breaking changes](https://github.com/rust-bitcoin/rust-bitcoin/pulls?q=is%3Apr+label%3A%22API+break%22+is%3Aclosed+milestone%3A0.29.0+)

## Highlights
```

Reply:
Agreed. And we need to add https://github.com/rust-bitcoin/rust-bitcoin/pull/1161 to the list

Reply:
Added.

Reply:
I like the list of breaking changes, it raises a question - if we are to use this sort of link (filtered search) for each release we have to ensure all PRs have a milestone when they merge. It would be nice if this was automated so we don't miss any.

Reply:
I've added the sections you suggest, thanks. Added #1161 to highlights also.

Reply:
Yep, this time I went over them and manually modified. I'd like to avoid repeating it...